bedrock/mozorg/templates/mozorg/internet-health/index.html

```diff
@@ -4,7 +4,7 @@
 
 {% extends "base-pebbles.html" %}
 
-{% add_lang_files "mozorg/internet-health/index" %}
+{% add_lang_files "mozorg/internet-health/shared" %}
```

Comment:
I'm not sure this will work, see: https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/views.py#L307-L310